### ISSUE TITLE
Make homepage update

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wordpress.org#367]

**Comment by adamwoodnz** [comment]
@WordPress/meta-design can we move ahead with the homepage design?

**Comment by jasmussen** [comment]
Yes, though please keep the existing team icons! You can put them in a circle or not, put them left of the title or not. 

Please be mindful of the correct accessibility of the particular collapsibles; if this ends up becoming a headache, it's okay to launch without the collapsibles, i.e. look much closer to the existing layout that's shipping at the moment.
